Apps/WebClient/src/ClientApp/src/store/modules/vaccinationStatus/types.ts

```diff
@@ -48,6 +49,9 @@ export interface VaccinationStatusGetters
     ): VaccinationStatus | undefined;
     isLoading(state: VaccinationStatusState): boolean;
     error(state: VaccinationStatusState): BannerError | undefined;
+    publicVaccineRecordError(
+        state: VaccinationStatusState
+    ): BannerError | undefined;
```

Comment:
Nit: might be better to move this after `publicVaccineRecordIsLoading` just for organizational purposes.